bin/varnishtest/vtc_varnish.c

```diff
@@ -271,7 +271,8 @@ varnishlog_thread(void *priv)
 		if (i == 0) {
 			/* Nothing to do but wait */
 			v->vsl_idle++;
-			if (!(VSM_Status(vsm) & VSM_WRK_RUNNING)) {
+			unsigned xxx = VSM_Status(vsm) & VSM_WRK_MASK;
```

Comment:
Looks a little unusual to see a variable declared here.

Reply:
Yes, I called it "xxx" to mean this was incomplete, because when I produced the original draft I rushed the varnishtest parts (still in a state of draft today).

The important things are to review the VSM status changes and share feedback on VSM status test coverage in varnishtest. Thank you for your review!

Reply:
Done.